phac-nml/irida-next
# Attachments: File format detection

## What does this PR do and why?
_Describe in detail what your merge request does and why._

This PR addresses issue #676

This update allows IRIDA Next to assign more common formats to file extensions

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Sign in
2. Select Projects on the sidebar and click on a project that is listed
3. Select Samples in the side bar and select one of the sample IDs
4. Upload files to the sample and check if the format matches the file extension
5. Repeat step 4 with any of the following files: 
    .gbf .gbf.gz .gbk .gbk.gz .genbank .genbank.gz
    .txt.gz .rtf.gz
    .json .json.gz
    .csv .csv.gz .tsv .tsv.gz

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
